src/sentry/static/sentry/app/views/settings/components/dataPrivacyRulesPanel/dataPrivacyRulesPanel.tsx

```diff
@@ -241,25 +241,32 @@ class ProjectDataPrivacyRulesPanel extends React.Component<Props, State> {
   };
 
   render() {
+    const {panelHeaderSubTitle} = this.props;
     const {rules, savedRules} = this.state;
     const hideButtonBar = savedRules.length === 0 && rules.length === 0;
     return (
       <React.Fragment>
         <Panel>
-          <PanelHeader>{t('Data Privacy Rules')}</PanelHeader>
-          <AlertLink
-            priority="info"
-            icon="icon-docs"
-            href="https://docs.sentry.io/data-management/advanced-datascrubbing/"
-            size="small"
-            openInNewTab
-            withoutMarginBottom
-          >
-            {t('Check out how to use advanced datascrubbing')}
-          </AlertLink>
+          <StyledPanelHeader>
+            {t('Data Privacy Rules')}
+            {panelHeaderSubTitle && (
+              <PanelHeaderSubTitle>{panelHeaderSubTitle}</PanelHeaderSubTitle>
+            )}
+          </StyledPanelHeader>
+          <AlertMessage
```

Comment:
I think you'll want `<PanelAlert>` here -- we should refactor AlertMessage and its storybook, I don't think it's to be used directly.

Reply:
oh Thanks...I didn't know about the `<PanelAlert>` component ... we should definitely add it in the Storybook